index.js

```diff
@@ -50,7 +51,7 @@ var ElectronBrowser = function(baseBrowserDecorator, args, electronOpts) {
       }],
       'exec': ['main.js:write', 'package.json:write', function(callback) {
         process.env.NODE_PATH = searchPaths.join(path.delimiter);
-        self._execCommand(self._getCommand(), [STATIC_PATH]);
+        self._execCommand(self._getCommand(), customFlags.concat([STATIC_PATH]));
```

Comment:
Array wrapper seems to be redundant now.

```suggestion
        self._execCommand(self._getCommand(), customFlags.concat(STATIC_PATH));
```

Reply:
True, I didn't know that _concat_ can do that. Thanks for pointing that out!
I updated it in my commit and pushed it again.